### DISCUSSION TITLE
Daemon not started yet / Failed to create the chia daemon

### DISCUSSION THREAD

**Discussion by 035uros** [Chia-Network/chia-blockchain#5273] [Support]
Hello,
I running into an issue while using chia on Linux RedHat without GUI. I started with creating plots but when I enter:

`chia start farmer wallet`

I get this error:

`OSError: [Errno 98] error while attempting to bind on address ('127.0.0.1', 55400): address already in use
Daemon not started yet
Failed to create the chia daemon`

Anyone can help?

Thank you!

**Comment by 035uros** [comment]
Update:

I found this solution and it worked! At least a part of the problem is gone, now this Timelord thing makes a problem.
I am a noob, is this necessary? Here is link to the solution I found: https://github.com/Chia-Network/chia-blockchain/issues/4153

` File "/mnt/chia-blockchain/venv/bin/chia_timelord_launcher", line 33, in <module>
    sys.exit(load_entry_point('chia-blockchain', 'console_scripts', 'chia_timelord_launcher')())
  File "/mnt/chia-blockchain/chia/timelord/timelord_launcher.py", line 107, in main
    loop.run_until_complete(spawn_all_processes(config, net_config))
  File "/usr/local/lib/python3.7/asyncio/base_events.py", line 587, in run_until_complete
    return future.result()
  File "/mnt/chia-blockchain/chia/timelord/timelord_launcher.py", line 85, in spawn_all_processes
    await asyncio.gather(*awaitables)
  File "/mnt/chia-blockchain/chia/timelord/timelord_launcher.py", line 45, in spawn_process
    path_to_vdf_client = find_vdf_client()
  File "/mnt/chia-blockchain/chia/timelord/timelord_launcher.py", line 39, in find_vdf_client
    raise FileNotFoundError("can't find vdf_client binary")
FileNotFoundError: can't find vdf_client binary`

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
